scripts/fix_raven_docstrings.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on May 22, 2018
+  @author: alfoa
+  This is a utility tool to fix the docstrings in RAVEN
+"""
+
+#For future compatibility with Python 3
+from __future__ import division, print_function, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+#End compatibility block for Python 3
+
+#External Modules--------------------
+import sys
+import os
+
+#External Modules--------------------
+
+
+def trimQuoteDocstring(docstring, quoteIndent, maxColumns=120):
+  """
+    This method is aimed to trim to the quote docstrings to the
+    be limited in the maxColumns number
+    @ In, docstring, str, the docstring that need to be modified
+    @ In, quoteIndent, int, the indentation number of columns of the docstring
+    @ In, maxColumns, int, optional, the maximum number of columns allowed (default 120)
+    @ Out, outputLines, list, the list of new lines that need to be inserted
+  """
+  lines = [elm.expandtabs() for elm in docstring]
+  outputLines = [" " * quoteIndent + '"""' + "\n"]
+  # find if "@" are present
+  inIndex = []
+  outIndex = []
+  for cnt, line in enumerate(lines):
+    if line.strip().startswith("@"):
+      if "@ in" in line.lower() or '@in' in line.lower():
+        inIndex.append(cnt)
+      elif "@ out" in line.lower() or '@out' in line.lower():
+        outIndex.append(cnt)
+  for cnt, line in enumerate(lines):
+    indentedLine = " " * (quoteIndent + 2) + line.lstrip()
+    if cnt in inIndex + outIndex:
+      indentedLine = indentedLine.replace("@Out", "@ Out").replace(
```

Comment:
a lot of lines like this ones have been split on multiple lines. is this really needed?

Reply:
unfortunately yes because they were longer that 120 columns